lib/models/beer.dart

```diff
@@ -24,16 +51,24 @@ class Beer {
       name: json['name'] as String,
       tagline: json['tagline'] as String,
       description: json['description'] as String,
-      imageURL: json['image_url'] as String,
+      imageURL: toImageUrlOrDefault(json['image_url']),
+      firstBrewed: toDateOrDefault(json['first_brewed']),
+      abv: "${toStringOrDefault(json['abv'] as num)}%",
+      ibu: toStringOrDefault(json['ibu'] as num),
+      targetOg: toStringOrDefault(json['target_og'] as num),
+      targetFg: toStringOrDefault(json['target_fg'] as num),
+      ebc: toStringOrDefault(json['ebc'] as num),
+      srm: toStringOrDefault(json['srm'] as num),
+      volume: volumeToString(json['volume']),
+      boilVolume: volumeToString(json['boil_volume']),
+      ph: toStringOrDefault(json['ph'] as num),
+      attenuationLevel: toStringOrDefault(json['attenuation_level'] as num),
+      foodPairing: (json['food_pairing'] as List)?.map((e) => e as String)?.toList(),
```

Comment:
```suggestion
      foodPairing: (json['food_pairing'] as List<String>)?.map((e) => e)?.toList(),
```
As tu essayés en faisant comme ça :
```suggestion
      foodPairing: (json['food_pairing'] as List<String>)?.toList(),
```

Reply:
ça marche pas (les deux solutions) : 
flutter: type 'List<dynamic>' is not a subtype of type 'List<String>' in type cast